guide/C/ch_invest.xml

```diff
@@ -1088,22 +1083,21 @@ Income
       <sect3 id="invest-buy-gross">
         <title>Example: Buying Shares with Gross Pricing</title>
 
-        <para>In this example you will purchase $5000 of IBM stock, with a commission of $100. First step will be
+        <para>In this example, you will purchase $5,000 of IBM stock, with a commission of $100. First step will be
           to create the stock account for IBM. The existing <emphasis>Expenses:Commission</emphasis>
           account will be used. If you wish to track commissions to the individual stock level an
-          additional sub-account would be necessary. E.g.
-          <emphasis>Expenses:Commission:IBM</emphasis>.
+          additional sub-account would be necessary, like <emphasis>Expenses:Commission:IBM</emphasis>.
```

Comment:
6f28b75a

Reply:
No comma. I think "for example" would be better than "like" here.

Note that this may not be correct. I think that the Advanced Portfolio Report is able to associate commission splits with the stock without a separate account. @mtalexander?

Reply:
The Advanced Portfolio Report has two ways to associate transactions for  dividends and other income and expenses with a stock.  You can either use an income or expense account which has the same leaf name as the stock account or you can put a dummy split for the stock account in the transaction.  This split should have both the shares and cost set to zero.   See the screenshot below for an example of each way.  In the December 2014 transaction the name of the income account is not relevant.  The stock account for Ford is named Assets:UBS:UBS Suzan's Account:Ford Motor Company.

![Dividend](https://user-images.githubusercontent.com/829809/126915042-cef01a36-864a-4eae-ad73-797aea972c07.jpg)

Reply:
@mtalexander Thanks, but the question is about breaking out commissions and fees in a buy/sell transaction. That obviously has a split with amount and value in the stock account already.

Reply:
Sorry, I misunderstood the question.

Yes, you're right, there is no need for a separate commission subaccount for each stock to track commissions that are recorded as part of a stock buy or sell transaction.  TheAdvanced Portfolio Report should associate them with the correct stock without that.

Reply:
TODO

Reply:
Remove
```
If you wish to track commissions to the individual stock level an
          additional sub-account would be necessary, like <emphasis>Expenses:Commission:IBM</emphasis>.
```

f3de93dc

Reply:
> The Advanced Portfolio Report has two ways to associate transactions for dividends and other income and expenses with a stock. You can either use an income or expense account which has the same leaf name as the stock account or you can put a dummy split for the stock account in the transaction. This split should have both the shares and cost set to zero. See the screenshot below for an example of each way. In the December 2014 transaction the name of the income account is not relevant. The stock account for Ford is named Assets:UBS:UBS Suzan's Account:Ford Motor Company.
> 
> ![Dividend](https://user-images.githubusercontent.com/829809/126915042-cef01a36-864a-4eae-ad73-797aea972c07.jpg)



> The Advanced Portfolio Report has two ways to associate transactions for dividends and other income and expenses with a stock. You can either use an income or expense account which has the same leaf name as the stock account or you can put a dummy split for the stock account in the transaction. This split should have both the shares and cost set to zero. See the screenshot below for an example of each way. In the December 2014 transaction the name of the income account is not relevant. The stock account for Ford is named Assets:UBS:UBS Suzan's Account:Ford Motor Company.
> 
> ![Dividend](https://user-images.githubusercontent.com/829809/126915042-cef01a36-864a-4eae-ad73-797aea972c07.jpg)

On a quick search I found only a short section in https://code.gnucash.org/docs/C/gnucash-guide/rpt_standardrpts.html#rpt_grp_assetsliabs about the report and the Option description in https://code.gnucash.org/docs/C/gnucash-help/report-classes.html#report-assets
@mtalexander Can you perhaps write more about it like

- Purpose (Difference against the normal/simple Portfolio report)
- Requirements
- Options still valid and compete?
- Output column description
- More to mention?
If docbook is too hard to write, a wiki page would be OK.

@jralls I wonder if we should group both portfolio reports (more?) under Investment in Docs and menu?